packages/test-utils/src/wrapper.js

```diff
@@ -531,6 +531,81 @@ export default class Wrapper implements BaseWrapper {
     this.vnode = this.vm._vnode
   }
 
+  /**
+   * Sets element value and triggers input event
+   */
+  setValue (value: any) {
+    const el = this.element
+
+    if (!el) {
+      throwError('cannot call wrapper.setValue() on a wrapper without an element')
+    }
+
+    const tag = el.tagName
+    const type = this.attributes().type
+    const event = 'input'
+
+    if (this.isVueComponent) {
```

Comment:
Why is this check here? I don't think the function needs to behave differently if the wrapper contains a vue instance

Reply:
Its unnecesary, will remove